lib/kube/proxy/forwarder.go

```diff
@@ -91,6 +91,9 @@ type ForwarderConfig struct {
 	// PingPeriod is a period for sending ping messages on the incoming
 	// connection.
 	PingPeriod time.Duration
+	// StrictImpersonationCheck specifies whether to fail when impersonation
+	// permissions of this forwarder can't be tested.
+	StrictImpersonationCheck bool
```

Comment:
Do we need to set a default value for this somewhere?

Reply:
The default is the zero value (false).
This will only be set to true in the new `kubernetes_service` (when it's ready https://github.com/gravitational/teleport/pull/4611).